Formula/reaver.rb

```diff
@@ -1,9 +1,9 @@
 class Reaver < Formula
   desc "Implements brute force attack to recover WPA/WPA2 passkeys"
-  homepage "https://code.google.com/archive/p/reaver-wps/"
-  url "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/reaver-wps/reaver-1.4.tar.gz"
-  sha256 "add3050a4a05fe0ab6bfb291ee2de8e9b8a85f1e64ced93ee27a75744954b22d"
-  license "GPL-2.0"
+  homepage "https://github.com/t6x/reaver-wps-fork-t6x"
+  url "https://github.com/t6x/reaver-wps-fork-t6x/releases/download/v1.6.6/reaver-1.6.6.tar.xz"
```

Comment:
This seems like a fork, but a bunch of other distros (e.g. Debian, Fedora) use it as the source too: https://repology.org/project/reaver/versions